StackExchange.Profiling/Timing.cs

```diff
@@ -81,21 +81,76 @@ public Timing(MiniProfiler profiler, Timing parent, string name, decimal? minSav
         /// Gets or sets All sub-steps that occur within this Timing step. Add new children through <see cref="AddChild"/>
         /// </summary>
         [DataMember(Order = 5)]
-        public List<Timing> Children { get; set; }
+        public IReadOnlyList<Timing> Children
```

Comment:
so v4 breaks if I use IReadOnlyList there. It breaks because protobuf that is used there doesn't support read only collections. There is an option to use IImmutableList instead (which is supported by protobuf) but then there would be a problem with Dictionary, there is no lightweight implementation of IImmutableDictionary available. 

I think the best way is to revert to using List and Dictionary, what do you think?